### PR TITLE
Refactoring some of the functions to make it easier to use other hypervi...

### PR DESCRIPTION
**Pull request by rnirmal**
...sors

### CODE REVIEW

File: scripts/functions_qemu [hunk 1]
```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+#
+# Additional functions that would mostly just pertain to a Ubuntu + Qemu setup
+#
+
+function install_packages_1() {
```
Comment (ed-) [inline scripts/functions_qemu]: why `_1`? Surely we can be a little more descriptive?

**Comment by ed-** [comment]
+1

File: scripts/functions_qemu (hunk 1 above)
Comment (hub-cap) [inline scripts/functions_qemu]: cuz _1 is first, duh ;P

File: scripts/redstack [hunk 2]
```diff
@@ -316,11 +302,11 @@ function install_devstack_code() {
     fi
     # Clones all of the code to where devstack expects it to be
     cd $PATH_DEVSTACK_OUTPUT
-    for project in glance horizon keystone nova python-keystoneclient python-novaclient python-quantumclient
```
Comment (hub-cap) [inline scripts/redstack:319]: Nice removal.

**Comment by hub-cap** [comment]
Very nice work @rnirmal +100

File: scripts/functions_qemu (hunk 1 above)
Comment (rnirmal) [inline scripts/functions_qemu]: cos I didn't bother to change what existed.. but yeah a better descriptive name works.. will change that

**Pull request merged**